tests/integration/test_kinesis.py

```diff
@@ -23,11 +23,13 @@ def assert_consumers(count):
 
         # create stream and assert 0 consumers
         client.create_stream(StreamName=stream_name, ShardCount=1)
+        sleep(1)
         assert_consumers(0)
 
         # create consumer and assert 1 consumer
         consumer_name = 'cons1'
         response = client.register_stream_consumer(StreamARN=stream_arn, ConsumerName=consumer_name)
+        sleep(1)
```

Comment:
kinesis-mock applies KINESIS_LATENCY to RegisterStreamConsumer actions so I added a sleep here.